### ISSUE TITLE
[Mod Conflict] Zombie and skeleton horses don't burn in daylight when any dimension mod is installed. 

### ISSUE THREAD

**Issue by The-Bernician-Lamb** [Serilum/.issue-tracker#519]
## **Information**
Minecraft version: 1.16.5
Forge version: 36.1.2
Environment: Singleplayer

Mod name: Skeleton Horse Spawn, Zombie Horse Spawn, Collective
Mod version: 1.6, 2.7, 2.25

## **Description**
When testing some of your mods for my latest modpack I noticed that zombie and skeleton horses were not burning in sunlight like were supposed to. 

After extensive testing I found that this was caused by having any dimension mod installed, which for some reason seems to prevent zombie and skeleton horses from burning in sunlight. This the only mod that seems to cause this and without this mod installed they burn in sunlight like they are supposed to.

Some screenshots for comparison:

Without Dimension mod:
![2021-03-25_16 20 14](https://user-images.githubusercontent.com/64381954/112511146-07280a80-8d8a-11eb-803b-cdf9bd6b8c3b.png)

With Dimension mod:
![2021-03-25_16 09 16](https://user-images.githubusercontent.com/64381954/112511177-1313cc80-8d8a-11eb-84a4-4c26f2a7eaec.png)

I've already opened an issue about it on their github which you can find here: https://github.com/AllenSeitz/DimDungeons/issues/26

## **Latest Log**
https://pastebin.com/areDpi7H
(No crash report as game doesn't crash)

**Comment by The-Bernician-Lamb** [comment]
Update: 

After doing some testing for the dev of Dimensional Dungeons I have found that this isn't specifically caused by his mod and is just caused by having any mod or datapack installed that adds a new dimension, which according to him could be the result of a vanilla bug. 

Therefore, I'm unsure of what you'll be able to do to address this problem if it is caused by something in vanilla.

**Comment by ricksouth** [comment]
Thanks for submitting the bug report and the detailed description! I've changed how the mods handle dimensions and keep track of the zombie/skeleton horses. Tested it with the latest version of Dimensional Dungeons and it should work as intended now. :)

In version 3.0 of https://www.curseforge.com/minecraft/mc-mods/zombie-horse-spawn/files/all
and version 1.9 of https://www.curseforge.com/minecraft/mc-mods/skeleton-horse-spawn/files/all.
